### PR TITLE
Only ranges dates if the total range is less than a year

### PR DESCRIPTION
**Pull request by fdansv**
Ref #455 

@javisantana

### CODE REVIEW

**Comment by javisantana** [comment]
sounds right but I'd try to add some tests here, looks like is becoming kind of hard to follow logic

**Comment by fdansv** [comment]
@javisantana added, all good?

**Comment by javisantana** [comment]
fdescribe ? what does it do?

**Comment by javisantana** [comment]
apart from that comment, all good

**Comment by fdansv** [comment]
Ah damn, good that you mentioned it!!! That's just for Jasmine to perform only that test, instead of the whole suite. Shouldn't have committed that, eeeesh.
